README.md

```diff
@@ -1,4 +1,5 @@
-# Large Hadron Migrator [![Build Status][5]][4]
+# Large Hadron Migrator
+[![Build status](https://badge.buildkite.com/6ed04595f04c5cf6f9f1453afd3705046f6c83088bd29cecf7.svg)](https://buildkite.com/shopify/lhm)
```

Comment:
May want to add a note about how this LHM is a downstream fork for our usages as this is a public repository.